tests/time_stack_tests.py

```diff
@@ -0,0 +1,137 @@
+'''
+Copyright 2015 Planet Labs, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+'''
+import unittest
+import numpy
+import os
+
+from osgeo import gdal
+
+from radiometric_normalization import time_stack
+
+
+class Tests(unittest.TestCase):
+    def test_organise_images_to_bands(self):
+        # Seven images of five bands of 10 by 11 pixels
+        all_images = []
+        nodata = 2 ** 15 - 1
+        for i in range(7):
```

Comment:
general convention is to use xrange for something like this, but it's not a big deal. I appreciate that you choose one option (range) and stick with it